py4web/core.py

```diff
@@ -1303,14 +1303,14 @@ def clear_modules():
                     del Reloader.MODULES[app_name]
                     clear_modules()
 
-                load_module_stdout = io.StringIO()
-                with redirect_stdout(load_module_stdout):
+                load_module_message = None
+                with io.StringIO() as buf, redirect_stdout(buf), redirect_stderr(buf):
```

Comment:
This appears to leak the stderr redirection into the loaded modules. rocket tries to write into buf after it is closed.